packages/cbioportal-utils/src/mutation/MutationAnnotator.ts

```diff
@@ -101,7 +101,8 @@ export function filterMutationByTranscriptId(
 export function getMutationToTranscriptId(
     mutation: Mutation,
     ensemblTranscriptId: string,
-    indexedVariantAnnotations: { [genomicLocation: string]: VariantAnnotation }
+    indexedVariantAnnotations: { [genomicLocation: string]: VariantAnnotation },
+    overwriteWithAnnotatedMutation?: boolean
```

Comment:
Ideally we should add new tests for the updated logic in this function and all the other functions modified in this utility file. Not sure if it is critical for this PR though